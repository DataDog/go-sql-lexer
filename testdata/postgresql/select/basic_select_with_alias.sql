SELECT u.id AS user_id, u.name AS username FROM users u;
