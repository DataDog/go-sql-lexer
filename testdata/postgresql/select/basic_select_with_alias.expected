[
    {
        "expected": "SELECT u.id, u.name FROM users u"
    },
    {
        "normalizer_config": {
            "keep_sql_alias": true
        },
        "expected": "SELECT u.id AS user_id, u.name AS username FROM users u"
    },
    {
        "normalizer_config": {
            "keep_trailing_semicolon": true
        },
        "expected": "SELECT u.id, u.name FROM users u;"
    }
]